### ISSUE TITLE
User does not have access to any teams with an active membership

### ISSUE THREAD

**Issue by eichert12** [HipByte/motion-provisioning#8]
I have configured motion-provisioning with the free:true flag, however, I'm still getting the following error message when trying to run my app:

"User AAA@bbb.com does not have access to any teams with an active membership"

I have a team in my account that expired and has not been renewed yet so I may have a bit of a unique situation that isn't appropriately handled.

```
/Users/steve/.rvm/gems/ruby-2.2.4@rubymotion/gems/spaceship-0.31.10/lib/spaceship/portal/portal_client.rb:70:in `team_id'
/Users/steve/.rvm/gems/ruby-2.2.4@rubymotion/gems/motion-provisioning-0.0.4/lib/motion-provisioning.rb:96:in `client'
/Users/steve/.rvm/gems/ruby-2.2.4@rubymotion/gems/motion-provisioning-0.0.4/lib/motion-provisioning/certificate.rb:6:in `client'
/Users/steve/.rvm/gems/ruby-2.2.4@rubymotion/gems/motion-provisioning-0.0.4/lib/motion-provisioning/certificate.rb:42:in `certificate_name'
/Users/steve/.rvm/gems/ruby-2.2.4@rubymotion/gems/motion-provisioning-0.0.4/lib/motion-provisioning.rb:195:in `certificate'
.../Rakefile:37:in `block (2 levels) in <top (required)>'
```

**Comment by MarkVillacampa** [comment]
Do you have the `free` flag set on both the certificate and the provisioning profile? 

If that's not the issue, can you run `VERBOSE=1 rake` (or variant) and paste the the response of the request to the following url?

```
>> POST: https://developerservices2.apple.com/services/QH65B2/listTeams.action
```

Make sure to anonymise any sensitive information.

**Comment by eichert12** [comment]
I have the following in my Rakefile

```
app.codesign_certificate = MotionProvisioning.certificate(
      type: :development,
      platform: :ios,
      free: true)

    app.provisioning_profile = MotionProvisioning.profile(
      bundle_identifier: app.identifier,
      app_name: app.name,
      platform: :ios,
      type: :development,
      free: true)
```

The response to the listTeams action is:

`{"teams"=>[], "creationTimestamp"=>"2016-09-27T13:12:48Z", "resultCode"=>0, "userLocale"=>"en_US", "protocolVersion"=>"QH65B2", "requestUrl"=>"https://developerservices2.apple.com:443/services/QH65B2/listTeams.action", "responseId"=>"Abc...."}`

When I go into Xcode the Development Team dropdown includes "Steve Eichert (Personal Team)".

**Comment by scally** [comment]
I'm also having this issue with my expired apple dev account. Just in case it was the problem, I also created a new apple ID and added it as a new free developer account, and still have this same issue, so it seems more likely related to free acounts and not just lapsed ones.

**Comment by eichert12** [comment]
I tried creating a new account as well to verify it wasn't an issue with lapsed accounts.  Glad (but also not) I'm not the only one running into this issue.  I was fearful it was a "user error". :)

**Comment by doutatsu** [comment]
I am getting the same error

**Comment by philomory** [comment]
I can confirm, this is an issue for me, as well

**Comment by andrewhavens** [comment]
This has been resolved in version 1.0.0.
